drivers/display/uc81xx.c

```
@@ -279,13 +274,9 @@ static int uc81xx_write(const struct device *dev, const uint16_t x, const uint16
 		}
 	}
 
-	if (config->override_cdi) {
-		/* Enable boarder output */
-		data->bdd_polarity &= ~UC81XX_CDI_BDZ;
-		if (uc81xx_write_cmd_uint8(dev, UC81XX_CMD_CDI,
-					   data->bdd_polarity)) {
-			return -EIO;
-		}
+	/* Enable boarder output */
+	if (config->quirks->set_cdi(dev, true)) {
+		return -EIO;
```

Why swallow the error instead of propagating it up? Here and above.